src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -4000,7 +4045,120 @@ subroutine cat_enkf_increments(                               &
        end do
        
        ! ----------------------------------
-       
+    case (11) select_update_type ! 1d snow (asnow) analysis; MODIS snow cover fraction jpark50
+    !=============================================================================
+    !jpark50: 28-Jul-2021: For asnow update:
+    !The increment defined here is a direct replacement; there is no use of EnkF
+    !=============================================================================
+    if (logit) write (logunit, *) 'get 1d snow(asnow) increments; MODIS SCF'
+  
+    do n_e=1,N_ens
```

Comment:
This call to "calc_asnow" appears to compute the forecast of the observation that are used below.  But this information should probably come from the "Obs_pred" (observation predictions), which have been calculated in "get_obs_pred()".  Note that "get_obs_pred()" was modified to compute "asnow" model forecasts, and "Obs_pred" is passed into this subroutine.